cameradummy:
  ros__parameters:
    img_dir: /home/finn/pem/stereovoxelnet_dataset/half_size/city
    topic_img_left: /left/image_raw
    topic_camera_left: /left/camera_info
    topic_img_right: /right/image_raw
    topic_camera_right: /right/camera_info
    frames_per_second: .5
    topic_disparity: /disparity